helper_functions.py

```diff
@@ -19,6 +19,19 @@ def commit_note_to_db(note_title, new_note):
     
     return note
 
+######'/notes/edit/<id>' ######
+def update_note(note_id, note_title, note_content):
+    """ Update a note By Id Return Updated Note Object """
+
+    # note = Note.query.filter_by(id=note_id).update(dict(title=note_title, content=note_content))
+    note = Note.query.filter_by(id=note_id).first()
+    note.title = note_title
+    note.content = note_content
+    
+   
+    db.session.commit()
```

Comment:
what happens if there's not a note with `id=note_id`?

do the changes actually get saved here?